client/components/Nav.js

```diff
@@ -62,12 +62,13 @@ class Nav extends Component {
           <div className='levels' onClick={() => this.refs.levelSelect.show()} >{'Level ' + this.props.currentLevel}</div>
           <SkyLight hideOnOverlayClicked={true} ref='levelSelect' dialogStyles={levelsModalStyle} >
             <div className='about-backstory'>
-              <p style={{fontSize: 20}} >Select a level to play</p>
+              <h3>Select a level to play</h3>
```

Comment:
Can you please classes .about-backstory and .modal-heading into this h3

import classNames from 'classnames'
className={classNames('modal-heading', 'about-backstory')}

chur